meetings/2023-08-23.md

```diff
@@ -41,16 +50,75 @@
+* SC: On a technical level, the proposed PR is removing that expectation on how a relationship should happen. The other issue is the process, 28 minutes to merging the PR, without the majority of CG even being aware, the chairs not approving, W3C staff not around. 
+* OO: I am reading the text. Does the WG decides something is mature enough or CG? I read it as WG is not allowed to take any new proposals until CG says it's okay.
+* SC: Rest of W3C documentation does not give that hint. WGs decide themselves, proposals can come from anywhere. If we're working on a spec in CG and w decide it's mature, it can propose it and WG can decide. If not in CG, anyone working outside the CG can go through the same process and propose, and WG will decide. 
+* OO: That is clear but the text does not reflect.
+* SC: There's a lot of process related information that is taken for granted. We can't spell out everything. Not saying every detail needs to be hidden. Current text does say WG makes the final call (on the proposal). 
+* PAC: I have no problem with the existing text (previous to the PR). If some people interpreted the wrong way maybe it's worth trying to fix. I agree with SC that a lot of knowledge is taken for granted but we should make it clearer if we can. It's up to W3C members to make the final call. WG decides to ask for recharters and AC approves. I don't think it's a good idea to remove the text completely as the PR did, but we could clarify. 
+* eP: I will post the link to these minutes on the PR thread and invite anyone with concerns to raise an issue. 
+* SC: Is there any dispute as to why that commit was reverted? 
+* OO: The process has changed a few times in the last few months. Might be worth writing down what the current one is. Barath made the first commit than we made extensive edits. CG started making edits around July. Barath did not know the process had changed.
+* SC: Which ones?
+* OO: As long as an editor approves it, you can merge.
+* SC: That did not incorporate the CG in any way. There was no deliberation. 
+* OO: Can we write down the process? 
+* SC: There's contributing guide but I will have a look. Just because there's a lack of clarity in the process does not entail that due diligence should not be followed on top of how we've been updating the charter through months of work in the open. 
+* OO: All I am saying is more clarity can't hurt because looking at history there have been a lot of merges like that in May, June.. 
+* WT: I would like to ask, we all agree clarity is good, but for now let's try to tackle another topic. 
```

Comment:
```suggestion
* SC: There's a contributing guide, but I will have a look. A lack of clarity in the process does not entail that due diligence should not be followed on top of how we've been updating the charter through months of work in the open. 
* OO: All I am saying is more clarity can't hurt, because looking at history, there have been a lot of merges like that in May, June...
* WT: I would like to ask, we all agree clarity is good, but for now let's try to tackle another topic. 
```